spec/api/v1/external_users/claims/final_claim_spec.rb

```diff
@@ -30,7 +28,7 @@
     }
   end
 
-  after(:all) { clean_database }
+  after { clean_database }
```

Comment:
I believe that `after(:all)` was done in places like this for efficiency. I would be in favour of avoiding `after(:all)` where possible but it is worth checking how long the tests are now taking to run in CircleCI.

Reply:
Fair point. Leaving it as `after(:all)` shaves about a minute off the tests which I think is enough to make it worthwhile. I've also changed it to `after(:all)` AGFS tests so that they're quicker too (and consistent).